projects/ngx-sub-form/src/lib/ngx-sub-form.component.ts

```diff
@@ -47,6 +43,24 @@ export abstract class NgxSubFormComponent<ControlInterface, FormInterface = Cont
     }, 0);
   }
 
+  private mapControls(
```

Comment:
This could be stronger typed with a generic inferred from the mapper

Reply:
Really not sure how to do that in that case :thinking: 
Would be glad if you could show me (tried a few things but they either makes things more verbose without adding value or don't work :sweat_smile:)

Reply:
Discussed IRL and done by @zakhenry in https://github.com/cloudnc/ngx-sub-form/pull/29